common/app/views/fragments/imageFigure.scala.html

```diff
@@ -36,7 +36,7 @@
         @if(isMain) {
             <meta itemprop="representativeOfPage" content="true">
         }
-        <meta itemprop="url" content="@ImgSrc.getFallbackUrl(picture)">
+        <meta itemprop="url" content="@ImgSrc.findNearestSrc(picture, Item700)">
```

Comment:
getFallbackUrl is found in Profile.scala. It returns the result of findNearestSrc, but with Item300 being passed in, just removing the middleman to get a bigger image